coalib/bearlib/abstractions/Linter.py

```diff
@@ -368,6 +379,37 @@ def process_output_corrected(self,
                              diffs={filename: diff},
                              severity=diff_severity)
 
+        def process_output_unidiff(self,
+                                   output,
+                                   filename,
+                                   file,
+                                   diff_severity=RESULT_SEVERITY.NORMAL,
+                                   result_message='Inconsistency found.'):
```

Comment:
Don't we need diff-distance ?
or showing whole in a single patch would be better ?